components/tooltip/index.jsx

```diff
@@ -59,6 +59,14 @@ const propTypes = {
 	 * Pass the one element that triggers the Tooltip as a child. It must be an element with `tabIndex` or an element that already has a `tabIndex` set such as an anchor or a button, so that keyboard users can tab to it.
 	 */
 	children: PropTypes.node,
+	/**
+	 * CSS classes to be added to the popover. That is the element with `.slds-popover` on it.
+	 */
+	className: PropTypes.oneOfType([
```

Comment:
Maybe we should be more specifc and expose this prop as `contentsClassName` since it's added to the dialog content div : `CSS classes to be added to the wrapping `div` of the contents of the dialog.`

Reply:
Good point. I used `className` to be consistent with the `Popover` component which uses the same prop name also for the dialog. Should it be called `dialogClassName` instead of `contentsClassname` since content represents the children in this component?